pom.xml

```diff
@@ -33,10 +32,17 @@
       <dependency>
         <groupId>org.jboss.arquillian</groupId>
         <artifactId>arquillian-bom</artifactId>
-        <version>1.7.0.Final</version>
+        <version>1.7.0.Alpha13</version>
```

Comment:
Is there really a need to use an alpha version? What happens when the stable version is used? Does something break?